manifests/t/TexasInstruments/TI-Nspire/CXPremiumTeacher/5.3.2.129/TexasInstruments.TI-Nspire.CXPremiumTeacher.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TexasInstruments.TI-Nspire.CXPremiumTeacher
+PackageVersion: 5.3.2.129
+PackageLocale: en-US
+Publisher: Texas Instruments

```

```suggestion
Publisher: Texas Instruments Inc.
```
This needs to match Control Panel.